ports/sese/vcpkg.json

```
@@ -0,0 +1,86 @@
+{
+  "name": "sese",
+  "version": "2.1.1",
+  "description": "a cross-platform framework",
```

```suggestion
  "description": "A cross-platform framework",
```
Not very descriptive, and the homepage doesn't help at all due to language barriers.